<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartChatDialog</class>
 <widget class="QDialog" name="StartChatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="chatroomInput">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>260</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="introLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>260</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Chatroom Prefix:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="introCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>260</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Set invitee as introducer</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
